app/docs/0.13.x/admin-api.md

```diff
@@ -2253,6 +2253,73 @@ HTTP 204 No Content
 
 ---
 
+### Show upstream health for node
+
+Displays the health status for all targets of a given upstream, according to
+the perspective of a specific Kong node. Note that, being node-specific
+information, making this same request to different nodes of the Kong cluster
+may produce different results. For example, one specific node of the Kong
+cluster may be experiencing network issues, causing it to fail to connect to
+some targets: these targets will be marked as as unhealthy to that node
+(directing traffic from that node to targets that it can successfully reach)
+but healthy to all others (which have no problems using that target).
+
+The `data` field of the response contains an array of target objects.
+The health for each target is returned in its `health` field:
+
+* If a target fails to be activated in the ring balancer due to DNS issues,
+  its status displays as `DNS_ERROR`.
+* When [health checks][healthchecks] are not enabled in the upstream
+  configuration, the health status for active targets is displayed as
+  `HEALTHCHECKS_OFF`.
+* When health checks are enabled and the target is determined to be healthy,
```

Comment:
Also when it is manually marked as healthy?

Reply:
Note the clever use of passive voice there: I didn't say who determined the target to be healthy. ;) But point taken, will fix!